go/vt/vtctl/workflow/framework_test.go

```diff
@@ -198,30 +198,38 @@ func (env *testEnv) addTablet(t *testing.T, ctx context.Context, id int, keyspac
 	return tablet
 }
 
-// addTableRoutingRules adds routing rules from the test env's source keyspace to
-// its target keyspace for the given tablet types and tables.
-func (env *testEnv) addTableRoutingRules(t *testing.T, ctx context.Context, tabletTypes []topodatapb.TabletType, tables []string) {
-	ks := env.targetKeyspace.KeyspaceName
-	rules := make(map[string][]string, len(tables)*(len(tabletTypes)*3))
+func (env *testEnv) saveRoutingRules(t *testing.T, rules map[string][]string) {
+	err := topotools.SaveRoutingRules(context.Background(), env.ts, rules)
+	require.NoError(t, err)
+	err = env.ts.RebuildSrvVSchema(context.Background(), nil)
+	require.NoError(t, err)
+}
+
+func (env *testEnv) updateTableRoutingRules(t *testing.T, ctx context.Context,
```

Comment:
env is the method receiver and it has the source and target keyspace names already. Do we want to be able to specify different keyspaces here?